src/Core/Content/ContactForm/SalesChannel/ContactFormRoute.php

```diff
@@ -117,6 +124,13 @@ public function load(RequestDataBag $data, SalesChannelContext $context): Contac
 
         $this->validateContactForm($data, $context);
 
+        $salutationCriteria = new Criteria([$data->get('salutationId')]);
```

Comment:
should we make sure, that this always returns a valid ID? or is ok, that in case of `null` all salutations are fetched?

Reply:
https://github.com/shopware/platform/blob/ac6b9b7dcd1f64d3a610cad98055f038daf71ef1/src/Core/Content/ContactForm/Validation/ContactFormValidationFactory.php#L53

Reply:
I think the validation assures that it's a valid id.